extensions/pubsub/pubsub.preview.runtime.ts

```diff
@@ -0,0 +1,58 @@
+import { PreviewAspect, PreviewRuntime } from '@teambit/preview';
+
+import { connectToParent } from 'penpal';
+
+import { BitBaseEvent } from './bit-base-event';
+import { PubsubAspect } from './pubsub.aspect';
+
+export class PubsubPreview {
```

Comment:
please add a comment saying this file will run in the preview iframe. I know it's deducible from filename, but I think an extra comment would help

Reply:
Done.